pkg/agent/openflow/client.go

```diff
@@ -828,12 +828,23 @@ func (c *client) InstallTraceflowFlows(dataplaneTag uint8) error {
 	for _, conj := range c.policyCache.List() {
 		for _, flow := range conj.(*policyRuleConjunction).metricFlows {
 			if flow.IsDropFlow() {
+				copyFlowBuilder := flow.CopyToBuilder(priorityNormal+2, false)
+				// Generate both IPv4 and IPv6 flows if the original drop flow doesn't match IP/IPv6
```

Comment:
Nit - add "." at end of the comment sentences.

Reply:
Updated. Also some nearby comments.